prow/tide/tide.go

```diff
@@ -685,3 +690,10 @@ type searchQuery struct {
 		}
 	} `graphql:"search(type: ISSUE, first: 100, after: $searchCursor, query: $query)"`
 }
+
+func (pr *PullRequest) SHA() string {
```

Comment:
can you put the commit message in as a comment here? there's a lot of subtle edge cases you're handling here!

Reply:
Done.